project/Dependencies.scala

```diff
@@ -11,21 +11,13 @@ object Dependencies {
 
   object V {
     // Scala
-    val cats             = "2.9.0"
     val catsEffect       = "3.5.0"
     val catsRetry        = "3.1.0"
-    val fs2              = "3.7.0"
-    val log4cats         = "2.6.0"
-    val http4s           = "0.23.15"
     val decline          = "2.4.1"
     val circe            = "0.14.1"
-    val circeConfig      = "0.10.0"
+    val http4s           = "0.23.15"
```

Comment:
There is now `0.23.23`

Reply:
Thanks, will bump everything to latest versions before 0.2.0 release.